src/Model/Position.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Model;
+
```

Comment:
Please add these use in the Model:

use Doctrine\ORM\Mapping\Entity;
use Doctrine\ORM\Mapping\Table;
use Doctrine\ORM\Mapping\Id;
use Doctrine\ORM\Mapping\Column;
use Doctrine\ORM\Mapping\GeneratedValue;